networks/suzuka/suzuka-full-node/src/partial.rs

```diff
@@ -113,7 +104,6 @@ impl SuzukaPartialNode<Executor> {
 			light_node_client,
 			settlement_manager,
 			commitment_events,
-			movement_rest,
```

Comment:
Meaning this also should be reverted.

Reply:
The whole MovementRest is not used. We can remove it.

Reply:
I remove the MovementRest crate.